### ISSUE TITLE
Cog disposals system needs a redesign

### ISSUE THREAD

**Issue by EthanQix** [space-wizards/space-station-14#31446]
## Description
The disposals system in Cog station is an overengineered nightmare and barely works.
- The 70+ tiles long, walled conveyor system is hard to understand at a glance, and hard to navigate when it's turned on.
- The disposals system control levers are mapped all over the place. One is in the jani closet and controls the recycler belt. One is in the south sorting room and controls the long S-shaped belt. The last one is at the space chute, east.
- If the trash is diverted to the sorting room, then the trash exiting the sorting room loops back into the sorting room entrance instead of going to the recycler.
- The recycler is mounted backwards, despite what it looks like on the screenshot. The trash stays stuck in front of it. Not sure if mapping/sprite/recycler issue.
- People wanting to go from Chapel to Engi through maints have to cross the (running) conveyor belts.

Long story short, this disposals setup needs to be shortened and straightened. It should fit on a single screen. The extra space could be used for abandoned maints rooms.

**Reproduction**
- Be janitor on Cog.
- Try to use disposals.
- Realize you need a map, a wrench, and a fair bit of running around to get it working.

**Screenshots**

![image](https://github.com/user-attachments/assets/bb2ea5df-9e4d-43b6-84b1-889339d038a9)

**Comment by JustCone14** [comment]
Possibly fixed in #31458?

**Comment by Spessmann** [comment]
fixed in #31458

**Comment by EthanQix** [comment]
Most excellent.
